@here/harp-mapview/lib/DepthPrePass.ts

```diff
@@ -133,6 +133,9 @@ export function createDepthPrePassMesh(mesh: THREE.Mesh): THREE.Mesh {
         mesh.material instanceof Array
             ? mesh.material.map(createDepthPrePassMaterial)
             : createDepthPrePassMaterial(mesh.material);
+    // if (!(depthPassMaterial instanceof Array)) {
+    //     mesh.customDepthMaterial = depthPassMaterial;
+    // }
 
     const depthPassMesh = new THREE.Mesh(depthPassGeometry, depthPassMaterial);
```

Comment:
Do we also render the buildings twice when rendering the shadow map? This should be avoided

Reply:
This mesh has both cast / receive shadows flags false, so I don't think so.

Reply:
fixed.